### PR TITLE
Dropdown Actions: Change the short link action to being copied to the clipboard

### PR DESCRIPTION
**Pull request by pablinos**
I kept clicking on this link, which takes me to the post and the window location is the full URL. I thought I'd see if I could change that behaviour so the short link is copied to the clipboard. I also added the shake animation to the icon.

I'm not sure whether the code is organised in the right way. I've added the CSS to the module, but the JS is in the main view for posts, which seems wrong. I'm happy to refactor as needed!

## Testing

- On a WordPress test site install the [P2-breathe theme](https://o2app.files.wordpress.com/2018/04/p2-breathe.zip), along with the current version of the O2 plugin (clone this repo to the plugins directory).
- View the site and click on the three dot menu at the top right of a post, followed by the Shortlink option. 
- Notice that this takes you to the post as you're visiting the short link.
- Now activate this version of the plugin by switching the branch to `add/clipboard-copy-on-shortlink`
- Click on the Shortlink option again, and the icon should animate as the short link is copied to the system clipboard.
- Have fun pasting your short link in a text editor, or wherever you like!

### CODE REVIEW

**Comment by mattsherman** [comment]
@pablinos What's the status of this PR? Is it still actively being worked on, or should it be closed?

**Comment by pablinos** [comment]
Hey @mattsherman! It's not being worked on as it needs review. Shall I see if someone else can help?

**Comment by mattsherman** [comment]
@pablinos If it is just waiting for review, I'd suggest pinging individuals who worked recently on the files and see if they can review the changes for you.

Some test instructions on the PR would help make it easier for somebody to do this.

**Comment by pablinos** [comment]
Good point about the testing instructions @mattsherman, I've just updated the description. I'll ping some other people for a review too.

File: js/views/post.js [hunk 1]
```diff
@@ -194,6 +196,39 @@ o2.Views.Post = ( function( $ ) {
...
+		onShortLinkClick: function( event ) {
+			var shortLink = event.target.href;
+			event.preventDefault();
+			event.stopPropagation();
+
+			this.copyToClipboard( shortLink );
+
+			var element = $( event.target );
+
+            element.addClass( 'clipboard-shake' );
```
Comment (ebinnion) [inline js/views/post.js]: There's an alignment issue here and in the next few lines.

**Comment by mattsherman** [comment]
@pablinos I'm going to move this back to Needs Review so we can have somebody from @Automattic/delta-samus test it before we merge this.

**Pull request merged**